cg/constants/priority.py

```diff
@@ -18,3 +14,16 @@ class SlurmQos(StrEnum):
     "production": SlurmQos.NORMAL,
     "development": SlurmQos.LOW,
 }
+
+
+class IntEnum(int, Enum):
+    def __str__(self) -> str:
+        return int.__str__(self)
+
+
+class Priority(IntEnum):
+    research: int = 0
```

Comment:
Should these be upper case since they are constants?

Reply:
According to https://namingconvention.org/python/constant-naming.html, yes. I'll and change it and sees what happens.

Reply:
Do you also suggest that the CLI-commands using priority should use "SCREAMING_SNAKE_CASE"? (e.g. `cg set family --priority EXPRESS`)

Reply:
as discussed in person with HS we'll keep the names lowercase for now to keep the priority lowercase when used as CLI-argument